app/marketing/views.py

```diff
@@ -724,7 +724,7 @@ def leaderboard(request, key=''):
 
     if amount:
         amount_max = amount[0][0]
-        top_earners = ranks.order_by('-amount')[0:3].values_list('github_username', flat=True)
+        top_earners = ranks.order_by('-amount')[0:5].values_list('github_username', flat=True)
```

Comment:
Are this changes related?

Reply:
think this PR is not going to be merged at the same time as leaderboard changes, so all content related to leaderboard need to be removed from this PR to avoid break things when deployed.

Reply:
fixed